tabs/applications-setup/browser-setup.sh

```diff
@@ -0,0 +1,151 @@
+#!/bin/sh -e
+
+. ../common-script.sh
+
+install_chrome() {
+    printf "%b\n" "Installing Google Chrome..."
+    case $PACKAGER in
+        apt-get)
+            wget https://dl.google.com/linux/direct/google-chrome-stable_current_amd64.deb
+            $ESCALATION_TOOL dpkg -i google-chrome-stable_current_amd64.deb
+            ;;
+        zypper)
+            $ESCALATION_TOOL ${PACKAGER} addrepo http://dl.google.com/linux/chrome/rpm/stable/x86_64 Google-Chrome
```

Comment:
```suggestion
            "$ESCALATION_TOOL" "$PACKAGER" addrepo http://dl.google.com/linux/chrome/rpm/stable/x86_64 Google-Chrome
```

Do this for every one.